test/framework/src/main/java/org/elasticsearch/test/ESIntegTestCase.java

```diff
@@ -906,7 +906,7 @@ public static List<String> getDataStreamBackingIndexNames(String dataStreamName,
         GetDataStreamAction.Response response = safeGet(
             client().execute(
                 GetDataStreamAction.INSTANCE,
-                new GetDataStreamAction.Request(TEST_REQUEST_TIMEOUT, new String[] { dataStreamName })
+                new GetDataStreamAction.Request(SAFE_AWAIT_TIMEOUT, new String[] { dataStreamName })
```

Comment:
Since `safeGet(...)` uses `SAFE_AWAIT_TIMEOUT`, I changed the timeout here to avoid confusion.